.env.example

```diff
@@ -9,6 +9,9 @@ BLOB_STORAGE_CONTAINER=
 AZURE_AISEARCH_ENDPOINT=
 AZURE_AISEARCH_KEY=
 AZURE_SEARCH_INDEX_NAME=
+AZURE_AISEARCH_SUMMARIES_KEY=
+AZURE_SEARCH_SUMMARIES_INDEX_NAME=
+AZURE_OPENAI_API_MODEL_NAME=
```

Comment:
Thanks for adding these to the example file, can you also add the key reference to appConfig/appConfig.snd3 and snd4 please. This file tells Helm to take the ADO pipeline values and use them to build the helm app